### ISSUE TITLE
"disease" vs "pathology"

### ISSUE THREAD

**Issue by lzehl** [HumanBrainProject/openMINDS_controlledTerms#7]
Should these terms be merged or should we keep them separate?

disease definition "a disorder of structure or function in a human, animal, or plant, especially one that produces specific symptoms or that affects a specific location and is not simply a direct result of physical injury."
-> "all" diseases as controlledTerms

pathology definition "the structural and functional deviation from the normal that constitute disease or characterize a particular disease"
-> general category, meaning "healthy", "diseased", "impaired", "abnormal"  as controlledTerms

OR merged under a single term with: "healthy" + "all" diseases as controlledTerms

If they are kept separate: should we add them both to subject (state) as well as tissue sample (state)?
We also ask currently for "functionalImpairments" e.g. "thumb missing at left hand" for subjects in addition. 
I wonder if that could be also merged in or just put under "generalRemarks" (free comment field for a subject to add peculiarities)

@UlrikeS91 @tgbugs what do you think?

**Comment by lzehl** [comment]
@UlrikeS91 & @tgbugs could this still be of interest (e.g. as separated properties for specimen?) or should we close it for now?

**Comment by UlrikeS91** [comment]
@lzehl, I think this can be closed. We discussed it back then and concluded that it is enough to have "disease" (+ "diseaseModel"). We removed "pathology" as a controlledTerm/terminology but renamed the property to "pathology". 
If I remember correctly, the idea was:
Pathology = healty --> leave blank (and/or specify under "additionalRemarks")
Pathology = diseased --> choose disease (model)
Pathology = impaired --> specify under "additionalRemarks" (or as disease whenever possible?) 
Pathology = abnormal --> specify under "additionalRemarks" (or as disease whenever possible?)
